target/linux/realtek/dts/rtl8380_netgear_gs110tpp-v1.dts

```diff
@@ -1,226 +1,10 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
-/dts-v1/;
 
-#include "rtl838x.dtsi"
+/dts-v1/;
```

Comment:
dts-v1 was wrong in this file beforehand, as it's already in rtl838x.dtsi. Just remove it.